### PR TITLE
Determine if required_if field is mandatory

### PR DESCRIPTION
**Pull request by FaZeRs**

### CODE REVIEW

File: src/Database/Traits/Validation.php [hunk 1]
```diff
@@ -407,6 +407,16 @@ public function isAttributeRequired($attribute)
             return $this->isAttributeRequired($requiredWith);
         }
 
+        if (strpos($ruleset, 'required_if') !== false) {
+            $data = $this->getValidationAttributes();
+            $requiredIf = substr($ruleset, strpos($ruleset, 'required_if') + 12);
```
Comment (LukeTowers) [inline src/Database/Traits/Validation.php]: What does the $ruleset string look like here? It seems to me from an initial look at it that 12 is an arbitrary number that is subject to breaking.

Comment (FaZeRs) [reply src/Database/Traits/Validation.php]: 'required_if:anotherfield,value' and $requiredIf variable here is 'anotherfield,value'

Comment (LukeTowers) [reply src/Database/Traits/Validation.php]: what if there is more than one rule in the rule set?

Comment (FaZeRs) [reply src/Database/Traits/Validation.php]: It wont break if there is rule set before this, but it will break if there is rule set after this.

**Comment by LukeTowers** [comment]
Please also provide unit tests for this; ideally unit test the whole isAttributeRequired method

**Comment by bennothommo** [comment]
Closing as it has been over a month since any activity on this occurred and we are trying to figure out what issues are still relevant. If this is still something that you would like to see through to fruition please respond and we can get the ball rolling.

**Pull request closed without merging**